release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,1010 @@
+number of API resources a cluster administrator must interact with and
+streamline the installation of new Operators on {product-title} 4.2.
+
+To work with OperatorHub in {product-title} 4.1, cluster administrators
+primarily interacted with OperatorSource and CatalogSourceConfig API resources.
+OperatorSources are used to add external datastores where Operator bundles are
+stored.
+
+CatalogSourceConfigs were used to enable an Operator present in the
+OperatorSource of your cluster. Behind the scenes, it configured an Operator
+Lifecycle Manager (OLM) CatalogSource so that the Operator could then be managed
+by OLM.
+
+To reduce complexity, OperatorHub in {product-title} 4.2 no longer uses
+CatalogSourceConfigs in the workflow of installing Operators. Instead,
+CatalogSources are still created as a result of adding OperatorSources to the
+cluster, however Subscription resources are now created directly using the
+CatalogSource.
+
+[discrete]
+[id="ocp-4-2-global-catalog-namespace"]
+==== Global catalog namespace change
+
+In {product-title} 4.1, the default global catalog namespace, where
+CatalogSources are installed by default, is
+`openshift-operator-lifecycle-manager`. Starting with {product-title} 4.2, this
+has changed to the `openshift-marketplace` namespace.
+
+If you have installed an Operator from OperatorHub on a {product-title} 4.1
+cluster, the CatalogSource is in the same namespace as the Subscription. These
+Subscriptions are not affected by this change and should continue to behave
+normally after a cluster upgrade.
+
+In {product-title} 4.2, if you install an Operator from OperatorHub, the
+Subscription created refers to a CatalogSource located in the new global
+catalog namespace `openshift-marketplace`.
+
+[discrete]
+[id="ocp-4-2-global-catalog-namespace-workaround"]
+===== Workaround for existing Subscriptions in the previous global catalog namespace
+
+If you have existing CatalogSources in the old
+`openshift-operator-lifecycle-manager` namespace, any existing Subscription
+objects that are referring to the CatalogSource will fail to upgrade, and new
+Subscription objects that are referring to the CatalogSource will fail to
+install.
+
+To workaround such upgrade failures:
+
+.Procedure
+. Move the CatalogSource object from the previous global catalog namespace to
+the `openshift-marketplace` namespace.
+
+[id="ocp-41-deprecated-features"]
+=== Deprecated features
+
+[discrete]
+[id="ocp-4-2-deprecation-of-service-catalog"]
+==== Deprecation of the Service Catalog, the Template Service Broker, the Ansible Service Broker, and their Operators
+
+In {product-title} {product-version}, the Service Catalog, the Template Service
+Broker, the Ansible Service Broker, and their Operators are deprecated. They
+will be removed in a future {product-title} release.
+
+The following related APIs will be removed in a future release:
+
+* `*.servicecatalog.k8s.io/v1beta1`
+* `*.automationbroker.io/v1alpha1`
+* `*.osb.openshift.io/v1`
```

Comment:
I think that we need to add
`brokertemplateinstance.template.openshift.io` 

This is only added when template-service-broker is installed.
@bparees can you ack that this is correct and that we should add it here?

Reply:
actually it looks like brokertemplateinstance is served out of the openshift apiserver, so i guess it'll still be present (but utterly useless w/ no TSB).

https://github.com/openshift/openshift-apiserver/tree/master/pkg/template/apiserver/registry/brokertemplateinstance

Reply:
Should we remove this type in 4.4 as well?